interview-questions.md

```diff
@@ -15,41 +15,41 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: In my point of view git is the software inside my personal computer and github is the cloud that stores the information. In LEARN ACADEMY I use git and github when working on pair programing trough class challenges.
 
-  Researched answer:
+  Researched answer: git is aversion control system that allows for the user to manage and keep track of code history. github is a cloud based source that serves as a host for git repository data.
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: In my experience the logical operators that will return a boolean value are: ===,an ==.
 
-  Researched answer:
+  Researched answer: boolean values are equality operator that compare one value to the other and compare both to say if the statement is tru or false.
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: An index is a number. A value is a word that is used to specify the content. In my understanding when working in an array. the name "zeus" sets a value and its index would be 1
 
-  Researched answer:
+  Researched answer:an index is a read only property that has the position of a regular expression within a string. values are strings arrays numbers an other sets of data that comprises value.
```

Comment:
This is a bit unclear...let's organize it some for your notes later. Index is the location of each value. We start the count with 0. The value is what is located at each index.